text/0013-code-formatters.md

```diff
@@ -0,0 +1,27 @@
+# Code formatters
+- Start Date: Mid 2016
+- RFC PR: https://github.com/CanopyTax/frontend-rfc/pull/13
+
+## Squad vs Chapter vs Frontend-wide
+Each squad may choose whether to use a code formatter and, if so, which one to use.
```

Comment:
Do we not have some formatting standards? For example: spaces v tabs?

Reply:
I was thinking that that would fall under an rfc for styling and linters instead of the one for code formatters.

Reply:
Do you think it makes sense for the rfc for styling/linting to be combined with code formatters? I was planning on separate but would feel fine either way